### ISSUE TITLE
/opt/graphite not available - puppet run fails

### ISSUE THREAD

**Issue by Holundermann** [echocat/puppet-graphite#362]
Hi!
When using the module to install puppet it fails during installation. It seems like that the module expects graphite to be located in /opt/graphite - but the python module is installed in /usr/local/lib/python2.7/dist-packages/opt/graphite.
Happens in Debian 9.4:

**Comment by Holundermann** [comment]
Content of /opt/graphite:

```
ls -lah /opt/graphite/
total 8.0K
drwxr-xr-x 2 www-data www-data 4.0K Mar 26 14:18 .
drwxr-xr-x 4 root     root     4.0K Mar 26 14:18 ..
```

find graphite:
```
root@xxx:/# find . -name 'graphite*'
./root/.cache/pip/wheels/7d/aa/2e/850a9f226a7529e033b5bacb3ddb5480839b15d9c4e014079a/graphite_web-0.9.15-cp27-none-any.whl
./etc/apache2/sites-available/graphite.conf
./etc/apache2/sites-enabled/graphite.conf
./etc/icinga2/features-available/graphite.conf
./usr/lib/python2.7/dist-packages/graphite_web-*-py2.7.egg-info
./usr/local/webapp/content/img/graphite_short.png
./usr/local/webapp/content/img/graphite.png
./usr/local/conf/graphite.wsgi.example
./usr/local/lib/python2.7/dist-packages/graphite_web-0.9.15.dist-info
./usr/local/lib/python2.7/dist-packages/opt/graphite
./usr/local/lib/python2.7/dist-packages/opt/graphite/webapp/graphite
./var/lib/graphite-web
./var/log/apache2/graphite-web
./srv/graphite.db
./opt/graphite
```

Error Message:

```
Notice: /Stage[main]/Main/Notify[role: monitoring, Connection successfully established]/message: defined 'message' as 'role: monitoring, Connection successfully established'
Notice: /Stage[main]/Graphite::Install/Package[python-pip]/ensure: created
Notice: /Stage[main]/Graphite::Install/Package[graphite-web]/ensure: created
Info: /Stage[main]/Graphite::Install/Package[graphite-web]: Scheduling refresh of Exec[carbon_hack]
Info: /Stage[main]/Graphite::Install/Package[graphite-web]: Scheduling refresh of Exec[gweb_hack]
Notice: /Stage[main]/Graphite::Install/Exec[carbon_hack]: Triggered 'refresh' from 1 event
Notice: /Stage[main]/Graphite::Install/Exec[gweb_hack]: Triggered 'refresh' from 1 event
Info: Class[Graphite::Install]: Scheduling refresh of Class[Graphite::Config]
Info: Class[Graphite::Install]: Scheduling refresh of Exec[Initial django db creation]
Info: Class[Graphite::Config]: Scheduling refresh of Exec[graphite-reload-systemd]
Info: Class[Graphite::Config]: Scheduling refresh of Exec[Initial django db creation]
Info: Class[Graphite::Config]: Scheduling refresh of Service[carbon-cache]
Error: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /opt/graphite/webapp/graphite/local_settings.py20180326-1186-fmu7jr.lock (file: /etc/puppetlabs/code/environments/develop/modules/graphite/manifests/config.pp, line: 173)
Error: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /opt/graphite/webapp/graphite/local_settings.py20180326-1186-fmu7jr.lock (file: /etc/puppetlabs/code/environments/develop/modules/graphite/manifests/config.pp, line: 173)
Wrapped exception:
No such file or directory @ dir_s_mkdir - /opt/graphite/webapp/graphite/local_settings.py20180326-1186-fmu7jr.lock
Error: /Stage[main]/Graphite::Config/File[/opt/graphite/webapp/graphite/local_settings.py]/ensure: change from 'absent' to 'file' failed: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /opt/graphite/webapp/graphite/local_settings.py20180326-1186-fmu7jr.lock (file: /etc/puppetlabs/code/environments/develop/modules/graphite/manifests/config.pp, line: 173)

```

**Comment by mzhaase** [comment]
Having the same issue. This did not happen with version 7.2.0 of the package.

**Comment by dagobert** [comment]
I have the same issue on Debian Stretch.

I figured out the pip package `carbon` and pip package` graphite-web` are installed in folder 

/usr/local/lib/python2.7/dist-packages/opt/graphite/

Notice that it seems that the install path is not set correctly. Sadly I have not figured out how to fix this in the module.

It would be great if the maintainer could help, since the module is really great and I have used it long time.
